dev-config.edn

```diff
@@ -29,5 +29,6 @@
  :extra-pages-path "./test-data/extra-pages"
  :application-deadline-days 4
  :enable-pdf-api true
+ :catalogue-is-public true
```

Comment:
the catalogue-is-public option should be added to `config-defaults.edn` along with documentation

you should probably also set `:catalogue-is-public true` in `dev-config.edn`

Reply:
`you should probably also set :catalogue-is-public true in dev-config.edn`  isnt it what I just did?

Reply:
ok, I was not aware about this one 'the catalogue-is-public option should be added to config-defaults.edn along with documentation', I will do it

Reply:
Sorry, I meant to say _in `test-config.edn`_!

Yeah, all our config variables are documented (and their defaults set) in config-defaults.edn. We should probably add that to the pull request template :)

Reply:
Thanks @opqdonut !

Reply:
Shouldn't the be that it's not public, and does not need to be in all these configs because default would be private?

Reply:
It can go either way. The default should probably be `false` in any case.

After that we can decide if we want to keep the option `true` in dev&test or not. If it's `false` in dev&test, we need a separate test case for it being `true`.

We can work on these later too, as long as this PR does something consistent it's fine by me.

Reply:
Also we need a task of taking it into use somewhere.